release-tests/test/6_array.test.js

```diff
@@ -0,0 +1,49 @@
+import { suite } from "uvu";
+import { cd } from "zx";
+import { NARGO_BIN } from "./utils/nargo.js";
+import "./utils/zx.js";
+
+const test = suite("nargo");
+
+// Helps detect unresolved ProcessPromise.
+let promiseResolved = false;
+process.on("exit", () => {
+  if (!promiseResolved) {
+    console.error("Error: ProcessPromise never resolved.");
+    process.exitCode = 1;
+  }
+});

```

Comment:
Future improvement is that you can actually just pass a flag to node that does this